.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,12 @@
 
 ## What is the purpose of the change
 
-XXXXX
+<!--
+If this PR fixes a GitHub issue, please add `fixes #<XXX>` or `closes #<XXX>`.Please refer to the documentation for more information:
```

Comment:
Sorry for being nitpicking, please add a space between the period of "`closes #<XXX>`." and "Please" in the next sentence.

Reply:
@tsunghanjacktsai Done